server/src/DBTablesAction.cc

```diff
@@ -1114,7 +1115,7 @@ string ActionsQueryOption::Impl::makeConditionTemplate(void)
 	// host_id;
 	const ColumnDef &colDefHostId = COLUMN_DEF_ACTIONS[IDX_ACTIONS_HOST_ID];
 	cond += StringUtils::sprintf(
-	  "((%s IS NULL) OR (%s=%%" PRIu64 "))",
+	  "((%s IS NULL) OR (%s=%%s))",
```

Comment:
No, here '%s' should be used because, because 
corresponding argument at ActionsQueryOption::getCondition()
is rhs(triggerInfo.hostIdInServer) whose return type is const char *.